{
	"id": "oval:com.redhat.cve:def:202145115",
	"version": "636",
	"class": "vulnerability",
	"metadata": {
		"title": "CVE-2021-45115 django: Denial-of-service possibility in UserAttributeSimilarityValidator (moderate)",
		"reference": [
			{
				"ref_id": "CVE-2021-45115",
				"ref_url": "https://access.redhat.com/security/cve/CVE-2021-45115",
				"source": "CVE"
			}
		],
		"description": "DOCUMENTATION: A resource-consumption flaw was found in django's UserAttributeSimilarityValidator, where it incurred significant overhead evaluating any submitted password that was artificially large relative to comparison values. A network attacker could exploit this flaw to cause a denial of service. \n            STATEMENT: In Red Hat OpenStack Platform, because the flaw's impact is lower and the impacted functionality is not directly used, no update will be provided at this time for the python-django20 package.",
		"advisory": {
			"from": "secalert@redhat.com",
			"severity": "Moderate",
			"updated": {
				"date": "2022-07-05"
			},
			"cve": [
				{
					"text": "CVE-2021-45115",
					"cvss3": "7.5/CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
					"cwe": "CWE-400",
					"href": "https://access.redhat.com/security/cve/CVE-2021-45115",
					"impact": "moderate",
					"public": "20220104"
				}
			],
			"affected": {
				"resolution": [
					{
						"state": "Affected",
						"component": [
							"python3-django"
						]
					},
					{
						"state": "Will not fix",
						"component": [
							"python-django",
							"python2-django"
						]
					}
				]
			},
			"affected_cpe_list": {
				"cpe": [
					"cpe:/a:redhat:satellite:6.9",
					"cpe:/a:redhat:satellite:6.9::el7",
					"cpe:/a:redhat:satellite_capsule:6.9",
					"cpe:/a:redhat:satellite_capsule:6.9::el7"
				]
			},
			"issued": {}
		},
		"affected": {}
	},
	"criteria": {
		"operator": "OR",
		"criterias": [
			{
				"operator": "AND",
				"criterias": [
					{
						"operator": "OR",
						"criterias": [
							{
								"operator": "AND",
								"criterions": [
									{
										"test_ref": "oval:com.redhat.cve:tst:201919844003",
										"comment": "python-django is installed"
									},
									{
										"test_ref": "oval:com.redhat.cve:tst:201919844004",
										"comment": "python-django is signed with Red Hat redhatrelease2 key"
									}
								]
							},
							{
								"operator": "AND",
								"criterions": [
									{
										"test_ref": "oval:com.redhat.cve:tst:201919844001",
										"comment": "python2-django is installed"
									},
									{
										"test_ref": "oval:com.redhat.cve:tst:201919844002",
										"comment": "python2-django is signed with Red Hat redhatrelease2 key"
									}
								]
							},
							{
								"operator": "AND",
								"criterions": [
									{
										"test_ref": "oval:com.redhat.cve:tst:202144420001",
										"comment": "python3-django is installed"
									},
									{
										"test_ref": "oval:com.redhat.cve:tst:202144420002",
										"comment": "python3-django is signed with Red Hat redhatrelease2 key"
									}
								]
							}
						]
					}
				],
				"criterions": [
					{
						"test_ref": "oval:com.redhat.cve:tst:20134120011",
						"comment": "Red Hat Enterprise Linux 7 is installed"
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:com.redhat.cve:tst:20134120012",
				"comment": "Red Hat Enterprise Linux must be installed"
			}
		]
	}
}
